tutorials/quantize.md

```diff
@@ -1,14 +1,32 @@
 # Quantize the model
 
-When `--precision bf16-true` or `--precision 16-true` is used, the model weights will automatically be converted and consume less memory.
-However, this might not be enough for large models or when using GPUs with limited memory.
+This document provides different strategies for quantizing the various models available in Lit-GPT to reduce GPU memory usage, which is useful for running larger models on certain GPU hardware. 
+
+**All the examples below were run on an A100 40GB GPU.**
 
 > **Note**:
 > Quantization is only supported with inference (generate and chat scripts).
 
+
 ### Baseline
 
-All the examples below were run on an A100 40GB GPU.
+It's useful to start with a baseline to have a reference point for memory savings via the various quantization methods.
+
+
+```bash
+python generate/base.py --checkpoint_dir checkpoints/tiiuae/falcon-7b --precision 32-true --max_new_tokens 256
+...
+Time for inference 1: 10.69 sec total, 23.95 tokens/sec.
+Memory used: 28.95 GB
+```bash
```

Comment:
It looks like "```bash" signals to start a new code block which results in a weird-looking code/text mixture.

<img width="1008" alt="Screenshot 2023-07-26 at 2 16 03 PM" src="https://github.com/Lightning-AI/lit-gpt/assets/58434077/fa136a60-ef92-4220-a1c7-48f28ebac067">

Reply:
Thanks, somehow my local markdown editor didn't complain about this but this is of course an issue! Submitted a PR to fix this and the other cosmetics you highlighted.